starter-code/pizza.js

```diff
@@ -1 +1,77 @@
+'use strict';
 // Write your Pizza Builder JavaScript in this file.
+
+// Produce jqeury objects (arrays) and store in var
+var $greenPeppers = $('.green-pepper');
+var $mushrooms = $('.mushroom');
+var $peps = $('.pep');
+
+// Produce jquery objects (arrays with length 1) and store in var
+var $greenPeppersBtn = $('.btn-green-peppers');
+var $pepsBtn = $('.btn-pepperonni');
+var $mushroomsBtn = $('.btn-mushrooms');
+
+// Bind button click to html element display/hide toggle
+$greenPeppersBtn.click(function(){
+  $greenPeppers.toggle(0, function(){
+    if($(this).is(':hidden')) {
+      $greenPeppersBtn.removeClass('active');
```

Comment:
This is fine, but instead of using a conditional, you can solve this problem with toggleClass.